lib/vault/docker-compose.yml

```diff
@@ -2,12 +2,7 @@ version: '3.7'
 
 services:
   vault:
-    build:
-      context: ${PWD}/lib/vault
-      args:
-        - VAULT_VERSION=1.9.3
-        - venv_python=${VENV_PYTHON}
-    image: ${DOCKER_REGISTRY}/vault:${VAULT_VERSION:-latest}
+    image: library/vault:latest
```

Comment:
Why is there Dockerfile if not being used? Also, pin vault version unless rolling updates is acceptable.

Reply:
the Dockerfile builds vault-runner.